### ISSUE TITLE
Vrmlogger: setting VRM mode to off: send dying wish to VRM

### ISSUE THREAD

**Issue by wiebeytec** [victronenergy/venus#1329]
We log the new data attribute `vrm` (`/Settings/Network/VrmPortal`) with the VRM mode (off, read-only, full). This way, VRM can see that a device is running in read-only or full mode.

When turning it off, we currently have a bit of a catch-22, because vrmlogger is needed to send the 'off' (`0`) value. We can make vrmlogger sends this config change on exit, before actually exiting. 

If daemon tools sends a proper `SIGTERM`, we can implement the exit handler as we want, and it should be easy to implement.

**Comment by jhofstee** [comment]
vrmlogger is not stopped though, it just no longer logs to the internet, it can still log to sd-cards etc. If I am not mistaken VRM knows if there are no new log records arriving, so I guess it will settle after some time, right?

**Comment by wiebeytec** [comment]
> If I am not mistaken VRM knows if there are no new log records arriving, so I guess it will settle after some time, right?

If a 'no data' alarm trigger is active, it will send out an e-mail after `<log interval> + x` has expired. This will already happen now when people disable logging to VRM. I just confirmed by disabling logging on a site.

Also, the current communication about 'vrm log mode' is also not perfect. The installation I just tested on already said it was disabled, with a timestamp of '2024-02-27 21:09:19', but it was actually enabled.

In other words, it's already not perfect.
